keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/ClusterDeployStrategy.kt

```diff
@@ -13,6 +13,10 @@ abstract class ClusterDeployStrategy {
 
   companion object {
     val DEFAULT_WAIT_FOR_INSTANCES_UP: Duration = Duration.ofMinutes(30)
+    const val RED_BLACK_STRATEGY = "red-black"
```

Comment:
Note, red-black has been.. "deprecated" in favor of standard terms "blue-green"

Reply:
Seeing lots of existing "red-black" terms in this repo. Going to check if they can be migrated and follow up